config/common.go

```diff
@@ -13,6 +15,12 @@ import (
 var (
 	ConstMinQuestionsCountPerChallenge = 100
 	ConstMinQuestionsCountPerShard     = 100
+	DbStatusURL string
+	DbGuardURL string
+	RdURL string
+	FoundPgStatusString bool
+	FoundPgGuardString bool
+	FoundDbRdString bool
```

Comment:
Not sure why these are declared here. They are test-only variables that should only exist in unit test files.

Reply:
okay.